CS5520FeinbergSpring2024/team-project-group19
# Profile activity

1. Implemented changes to store user data in Firebase using a unique identifier.
2. Created a Profile Activity allowing users to view their ID and upload a profile picture.
3. Profile pictures are encoded to base64 and stored in Firebase if no picture exists.
4. Modified code to decode URL and display profile pictures if they exist for the current user.
5. Miscellaneous changes to handle user storage in Firebase.

Testing:
- Create a new user and ensure a unique identifier exists for each user.
- Click on the profile icon to open the Profile Activity and verify that the user ID is displayed.
- Add a photo or take a photo and see if it gets displayed and stored in Firebase.
- Log out and log back in to see if the profile picture still exists.
